### ISSUE TITLE
Page title and meta information is not accurate for checkout view

### ISSUE THREAD

**Issue by spencern** [reactioncommerce/reaction#3732]
### Expected behavior
When navigating the app, the page title and other meta information should be dynamically updated. When I browse to the checkout page, I should see something about the checkout in the page title.

### Actual behavior
When visiting the checkout section, the page title seems to get stuck on the previous product that was viewed.

<img src="http://g.recordit.co/hVht2KfcFS.gif">

### Steps to reproduce the behavior
1. Browse to the basic product
2. Add product to cart
3. Click the slide out cart notification to checkout
4. Observe the page title at checkout is identical to the previous PDP you visited

### Versions
```
Node: 8.9.0
NPM: 5.6.0
Meteor Node: 8.9.3
Meteor NPM: 5.5.1
Reaction CLI: 0.27.0
Reaction: 1.8.0
Reaction branch: release-1.8.0
Docker: 17.12.0-ce
```

**Comment by zenweasel** [comment]
Related to #3767 so taking this as well

**Comment by zenweasel** [comment]
Closed via #3804
